lib/rouge.rb

```diff
@@ -2,6 +2,15 @@
 
 # stdlib
 require 'pathname'
+require_relative 'rouge/version'
```

Comment:
Given the [.travis.yml](https://github.com/jneen/rouge/blob/master/.travis.yml) seems to test a minimum version of Ruby v1.9.3, I don't think this should be a problem.